packages/components/src/components/form/InputStyles.ts

```diff
@@ -98,7 +98,7 @@ export const baseInputStyle = css<BaseInputProps>`
         css`
             background: ${({ theme }) => theme.BG_GREY};
             box-shadow: none;
-            color: ${({ theme }) => theme.TYPE_DARK_GREY};
+            color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
```

Comment:
Shouldn't be a button for now, I suppose? It is because of this:

```
<Button
    variant="tertiary"
    // icon={revision ? 'EXTERNAL_LINK' : undefined}
    // alignIcon="right"
    disabled // TODO: this should link to an article in knowledge base or guide in the future
>
```

Reply:
dad90c9f640f4efd44175ba250ecfb8088ab2a92